search-cache-pipeline.yml

```diff
@@ -15,160 +18,166 @@ parameters:
-  displayName: Run Cache Updater
-
-- task: CopyFiles@2
-  inputs:
-    SourceFolder: $(System.DefaultWorkingDirectory)/NugetDownloadDirectory/SearchCache/
-    Contents: | 
-      NuGetTemplateSearchInfo*.json
-      nonTemplatePacks.json
-    TargetFolder: $(System.DefaultWorkingDirectory)/ArtifactsToPublish/
-
-- publish: $(System.DefaultWorkingDirectory)/ArtifactsToPublish/
-  artifact: outputs
-  displayName: Publish Artifacts
-
-- ${{ if eq(parameters.publishToBlob, true) }}:
-  - script: '$(Build.SourcesDirectory)/eng/SearchCache/InstallAzureCLI.sh'
-    displayName: Install Azure CLI
-
-  - bash: az config set extension.use_dynamic_install=yes_without_prompt
-    displayName: Disable Azure CLI prompts
-
-  - bash: >
-      az storage azcopy blob upload 
-      -c $(CacheFileStorageContainer)
-      --account-name $(CacheFileStorageAccount) 
-      -s '$(System.DefaultWorkingDirectory)/ArtifactsToPublish/NuGetTemplateSearchInfoVer2.json' 
-      --sas-token "$(CacheFileStorageSasToken)"
-      -d NuGetTemplateSearchInfoVer2.json
-      | tee upload.log
-      && grep ".*Number of Transfers Completed: 1" upload.log || (echo ; echo "Cache file upload failed"; false)
-    displayName: Upload to blob storage
-
-  - bash: >
-      az storage azcopy blob upload 
-      -c $(LegacyCacheFileStorageContainer)
-      --account-name $(LegacyCacheFileStorageAccount) 
-      -s '$(System.DefaultWorkingDirectory)/ArtifactsToPublish/NuGetTemplateSearchInfo.json' 
-      --sas-token "$(LegacyCacheFileStorageSasToken)"
-      -d NuGetTemplateSearchInfo.json
-      | tee upload-legacy.log
-      && grep ".*Number of Transfers Completed: 1" upload-legacy.log || (echo ; echo "Legacy cache file upload failed"; false)
-    displayName: Upload legacy file to blob storage
-
-  - bash: >
-      az storage azcopy blob upload 
-      -c $(NonTemplatePacksFileStorageContainer)
-      --account-name $(NonTemplatePacksFileStorageAccount) 
-      -s '$(System.DefaultWorkingDirectory)/ArtifactsToPublish/nonTemplatePacks.json' 
-      --sas-token "$(NonTemplatePacksFileStorageSasToken)"
-      -d nonTemplatePacks.json
-      | tee upload-non-template-packs.log
-      && grep ".*Number of Transfers Completed: 1" upload-non-template-packs.log || (echo ; echo "Legacy cache file upload failed"; false)
-    displayName: Upload non template packages file to blob storage
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: NetCore1ESPool-Internal
+        image: 1es-windows-2022
+        os: windows
+    pool:
+      name: NetCore1ESPool-Internal
+      image: 1es-ubuntu-2204
+      os: linux
+    customBuildTags:
+    - ES365AIMigrationTooling
```

Comment:
We don't need this. It is tracking information for using the conversion tool, which we didn't use here.